src/swarm-scripts/71.setup-auth-service.sh

```
@@ -0,0 +1,87 @@
+
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)